test/test-component.js

```diff
@@ -110,6 +110,15 @@ describe('Simple Component instance', function() {
       expect(el.shadowRoot).to.be.ok;
     });
 
+    it('successfully finds the panel root', function() {
+      el.attachedCallback();
+      childEl = document.createElement('nested-child');
+      childEl.setAttribute('panel-parent', el.panelID);
+      el.el.appendChild(childEl);
+      childEl.attachedCallback();
+      expect(childEl.$panelRoot).to.equal(el);
```

Comment:
So this tests a child element finding a parent with shadow dom. I'd say it's also worth testing that a child element with shadow dom successfully finds a parent, what do you think?